counsel.el

```diff
@@ -2985,11 +2985,11 @@ NEEDLE is the search string."
        (ivy-more-chars))
      (let* ((default-directory (ivy-state-directory ivy-last))
             (regex (counsel--grep-regex search-term))
-            (switches (concat (car command-args)
-                              (counsel--ag-extra-switches regex)
-                              (if (ivy--case-fold-p string)
+            (switches (concat (if (ivy--case-fold-p string)
                                   " -i "
-                                " -s "))))
+                                " -s ")
+                              (counsel--ag-extra-switches regex)
+                              (car command-args))))
```

Comment:
Have you tested whether this ordering continues to work when `counsel--ag-extra-switches` returns non-`nil`?

Reply:
It seems to.  `command--ag-extra-switches` only ever returns nil and `" --pcre2 "`, AFAICT.

Reply:
Right, thanks.